BDArmory/Weapons/ModuleWeapon.cs

```diff
@@ -1644,6 +1645,7 @@ public void PAWRefresh()
                 Fields["defaultDetonationRange"].guiActiveEditor = true;
                 Fields["detonationRange"].guiActive = true;
                 Fields["detonationRange"].guiActiveEditor = true;
+                detonationRange = -1;
```

Comment:
This causes the proximity range to reset when loading or spawning a craft, which `SetInitialDetonationDistance` then sets to the default (100 for bullets or 2/3 blast radius for rockets — should this also default to 100?), which I don't think is the intended behaviour.